app/src/pages/OnDeviceDisplay/ProtocolDetails/index.tsx

```diff
@@ -205,21 +235,24 @@ const Summary = (props: {
 
 interface ProtocolSectionContentProps {
   protocolId: string
-  protocolData: ProtocolResource
+  protocolData: Protocol | null | undefined
   currentOption: TabOption
 }
-const ProtocolSectionContent = (
-  props: ProtocolSectionContentProps
-): JSX.Element => {
-  const { protocolId, protocolData, currentOption } = props
-  let protocolSection
+const ProtocolSectionContent = ({
+  protocolId,
+  protocolData,
+  currentOption,
+}: ProtocolSectionContentProps): JSX.Element => {
+  if (protocolData === null || protocolData === undefined) return <></>
+
+  let protocolSection = null
```

Comment:
nit, you can type case protocolSection here
```suggestion
  let protocolSection : JSX.Element | null = null
```